FreeRTOS/FreeRTOS-Kernel
# Fix API for NULL task parameter

Description
-----------
This PR addresses the fix, when NULL Task Handle is passed as parameter to the ulTaskGetRunTimeCounter and ulTaskGetRunTimePercent APIs.

Test Steps
-----------
Run [CORTEX_MPU_M7_NUCLEO_H743ZI2_GCC_IAR_Keil](https://github.com/FreeRTOS/FreeRTOS/tree/main/FreeRTOS/Demo/CORTEX_MPU_M7_NUCLEO_H743ZI2_GCC_IAR_Keil)Demo with configGENERATE_RUN_TIME_STATS  = 1.

Checklist:
----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
